addons/l10n_fr_pos_cert/models/account_bank_statement.py

```diff
@@ -0,0 +1,23 @@
+from openerp import models, api
+from openerp.tools.translate import _
+from openerp.exceptions import UserError
+
+BKN_NO_TOUCH = _('You cannot modify anything on a %s (name: %s) that was created by point of sale operations.')
+
+
+class AccountBankStatement(models.Model):
+    _inherit = 'account.bank.statement'
+
+    def unlink(self):
+        for statement in self.filtered(lambda s: s.company_id._is_accounting_unalterable() and s.journal_id.journal_user):
```

Comment:
We wanted the inability to alter any bank statement or bank statement line but it is too complex (there are so many writes on those, even when opening/re-opening a POS session)

Here, we just prevent any deletion

Reply:
I think that it will generate bad side effect, as said above, for journals used for PoS AND invoices.
But well, it can be fixed later.

Could you just change your code, calling a function like this ?

```
@api.multi
def _check_accounting_unalterable(self):```

It will more easily to change / alter the behaviour of this test. 
regards.